libs/platform/src/lib/table/components/table-header-row/table-header-row.component.ts

```diff
@@ -141,9 +143,16 @@ export class TableHeaderRowComponent extends TableRowDirective implements OnInit
     _columnTrackBy(index: number, column: TableColumn): string {
         return column.name;
     }
+}
+
+/** @hidden */
+@Pipe({ name: 'isColumnHasHeaderMenu' })
+export class IsColumnHasHeaderMenuPipe implements PipeTransform {
```

Comment:
This sounds weird :D

Reply:
I agree, it was the name of the previous function